src/Util.ts

```diff
@@ -441,107 +441,104 @@ class Util extends BITBOXUtil {
           throw new Error(`utxo ${i} does not have a txid property.`)
       }
 
-      // Create an array of txid strings to feed to validateTxid
-      const txids = utxos.map((x: any) => x.txid)
-
-      // Validate the array of txids.
-      const validations: any = await this.validateTxid(txids)
-      // console.log(`validations: ${JSON.stringify(validations, null, 2)}`)
+      // Output array
+      const outAry = []
 
-      const outAry = [] // Output Array
-
-      // Loop through each element and compute final validation on any that
-      // returned true.
-      // Loop on utxos, not validations since txids can be the same and
-      // UTXOs are differentiated by vout value.
+      // Loop through each utxo
       for (let i = 0; i < utxos.length; i++) {
-        const thisUtxo = utxos[i]
-        // console.log(`thisUtxo: ${JSON.stringify(thisUtxo, null, 2)}`)
+        const utxo = utxos[i]
+        // console.log(`utxo: ${JSON.stringify(utxo,null,2)}`)
+
+        // Get raw transaction data from the full node and attempt to decode
+        // the OP_RETURN data.
+        // If there is no OP_RETURN, mark the UTXO as false.
+        let slpData: any = false
+        try {
+          slpData = await this.decodeOpReturn(utxo.txid)
+          // console.log(`slpData: ${JSON.stringify(slpData, null, 2)}`)
+        } catch (err) {
+          // console.log(`decodeOpReturn err: `, err)
 
-        // Find the validation entry that matches the current UTXO.
-        const thisValidation = validations.filter((x: any) => {
-          if(x !== null)
-            return x.txid === thisUtxo.txid
-          return false
-        })
-        // console.log(
-        //   `thisValidation: ${JSON.stringify(thisValidation, null, 2)}`
-        // )
-
-        // If the utxo is not SLP, then skip the loop.
-        if (!thisValidation[0] || !thisValidation[0].valid) {
-          outAry[i] = false
-          continue
-        }
+          if (
+            err.message === "Not an OP_RETURN" ||
+            err.message === "Not a SLP OP_RETURN"
```

Comment:
Is it possible to give these errors a code so we don't need to compare directly to the `message` string?  

Error message strings should be modifiable without breaking other code => These comparisons will eventually break from an unrelated change.

Reply:
With an error code, we'd need to do some sort of lookup in order to give the user an actionable error message. I'm not sure the best way to approach this.

Reply:
Also, these error codes are being generated within the same Util.ts library (`decodeOpReturn()`). They aren't coming from some external source, so unrelated changes breaking this flow is not a serious concern in my mind. A change would first have to break the unit tests.